docroot/themes/humsci/humsci_basic/sass-specs/_tools.mixins.themes.scss

```diff
@@ -0,0 +1,133 @@
+      @include assert {
+        @include output {
+          @include colorful {
+            color: green;
+          }
+        }
+        @include contains {
+          color: green;
+        }
+      }
+    }
+
+    @include it('does not return styles because the mixin does not match the $current-theme') {
+      @include assert {
+        @include output {
+          @include airy {
+            color: yellow;
+          }
+        }
+        @include expect { }
+      }
+    }
+  }
+
+  // Airy
+  @include describe('airy() mixin') {
+    $current-theme: 'airy' !global;
+
+    @include it('returns some properties specific to Airy') {
+      @include assert {
+        @include output {
+          @include airy {
+            color: yellow;
+          }
+        }
+        @include contains {
+          color: yellow;
+        }
+      }
+    }
+
+    @include it('does not return styles because the mixin does not match the $current-theme') {
+      @include assert {
+        @include output {
+          @include traditional {
+            color: purple;
+          }
+        }
+        @include expect { }
+      }
+    }
+  }
+
+  // Traditional
+  @include describe('traditional() mixin') {
+    $current-theme: 'traditional' !global;
+
+    @include it('returns some properties specific to Traditional') {
+      @include assert {
+        @include output {
+          @include traditional {
+            color: purple;
+          }
+        }
+        @include contains {
+          color: purple;
+        }
+      }
+    }
+
+    @include it('does not return styles because the mixin does not match the $current-theme') {
+      @include assert {
+        @include output {
+          @include colorful {
+            color: green;
+          }
+        }
+        @include expect { }
+      }
+    }
+  }
+
+  // Multi-theme
+  @include describe('Multi-theme themes() mixin') {
+    $current-theme: 'colorful' !global;
+
+    @include it('returns some properties specific to Colorful') {
+      @include assert {
+        @include output {
+          @include themes(('colorful', 'traditional')) {
+            color: brown;
+          }
+        }
+        @include contains {
+          color: brown;
+        }
+      }
+    }
+
+    @include it('does not return styles because the mixin does not include a reference to the $current-theme') {
+      @include assert {
+        @include output {
+          @include themes(('airy', 'traditional')) {
+            color: orchid;
+          }
+        }
+        @include expect { }
+      }
+    }
+
+    @include it('returns some properties specific to colorful theme when no params passed') {
+      // by default, it will just apply styles to all themes without params
+      @include assert {
+        @include output {
+          @include themes() {
+            color: papayawhip;
+          }
+        }
+        @include expect {
+          color: papayawhip;
+        }
+      }
+    }
+  }
+}
```

Comment:
Great test coverage with these tests! I also love the naming convention to match the file name of those mixins being tested. Could we add an empty line at the end of the file? Linters usually complain about that.

Reply:
I might need to make sure this is getting linted as well

Reply:
hmm actually scss rules for our sass-true tests don't align, so I won't include that in the lint task